packages/jest-docblock/src/__tests__/index.test.ts

```diff
@@ -119,48 +104,45 @@ describe('docblock', () => {
 
   it('parses >=3 of the same directives out of a docblock', () => {
     const code =
-      '/**' +
-      EOL +
-      '' +
-      ' * @x foo' +
-      EOL +
-      '' +
-      ' * @x bar' +
-      EOL +
-      '' +
-      ' * @x baz' +
-      EOL +
-      '' +
-      ' */';
+      String(
+        String(
+          String(String('/**' + EOL) + ' * @x foo' + EOL) + ' * @x bar' + EOL,
+        ) +
+          ' * @x baz' +
+          EOL,
+      ) + ' */';
     expect(docblock.parse(code)).toEqual({
       x: ['foo', 'bar', 'baz'],
     });
   });
 
   it('parses directives out of a docblock with comments', () => {
     const code =
-      '/**' +
-      EOL +
-      '' +
-      ' * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.' +
-      EOL +
-      '' +
-      ' * @team foo' +
-      EOL +
-      '' +
-      ' * @css a b' +
-      EOL +
-      '' +
-      ' *' +
-      EOL +
-      '' +
-      ' * And some license here' +
-      EOL +
-      '' +
-      ' * @preserve-whitespace' +
-      EOL +
-      '' +
-      ' */';
+      String(
```

Comment:
this looks like a pure mess :)

I think what you need here is to just wrap `EOL` in String wrapper
like `String(EOL)` and leave the rest as is